core/src/main/scripts/migrate_db.py

```diff
@@ -237,6 +238,48 @@ def check_and_remove_invalid_foreign_keys(cursor):
                 print('Invalid foreign key has been deleted.', file=OUTPUT_FILE)
     except MySQLdb.Error as msg:
         print(msg, file=ERROR_FILE)
+#since the foreign key can exist as sample_ibfk_1 or sample_ibfk_2
+#using the script to check for either name with reference to table and remove the one that exists
+def check_and_remove_foreign_keys_sample_ibfk(cursor):
```

Comment:
```suggestion
def check_and_remove_type_of_cancer_id_foreign_key(cursor):
"""The TYPE_OF_CANCER_ID foreign key in the sample table can be either sample_ibfk_1 or sample_ibfk_2. Figure out which one it is and remove it"""
```

Reply:
in the future there might be other foreign keys in sample, so maybe good to be slightly more explicit

Reply:
maybe even better: `check_and_remove_type_of_cancer_id_foreign_key_in_sample_table`